ckan/model/__init__.py

```diff
@@ -207,12 +208,18 @@ def init_db(self) -> None:
 
     def clean_db(self) -> None:
         self.commit_and_remove()
-        meta.metadata = MetaData(self.metadata.bind)
+        meta.metadata = MetaData()
+
+        if not meta.engine:
+            raise CkanConfigurationException("Model is not initialized")
```

Comment:
Can we add a small tip on how to fix this error in the message? I would not know what to do or where to check if I get that message.